### PR TITLE
make an html table to look at the next 100 predictions by arrival time

### PR DESCRIPTION
**Pull request by sghiaa**
[[3] A user can see recent predictions](https://app.asana.com/0/584764604969369/872531721286022)

this encompasses all the subtasks sorry gabe :/

### CODE REVIEW

File: config/test.exs [hunk 1]
```diff
@@ -13,6 +13,7 @@ config :logger, level: :warn
 config :prediction_analyzer, Predictions.Repo,
   username: System.get_env("DATABASE_POSTGRESQL_USERNAME") || "postgres",
   password: System.get_env("DATABASE_POSTGRESQL_PASSWORD") || "postgres",
+  adapter: Ecto.Adapters.Postgres,
```
Comment (losvedir) [inline config/test.exs]: Is this necessary? I think this is specified in `config.exs`.

Comment (sghiaa) [reply config/test.exs]: i was getting it complaining about this, but its possible taht i fixed the complaints in another way - ill remove this if it allows me to

File: lib/prediction_analyzer_web/templates/predictions/index.html.eex [hunk 2]
```diff
@@ -0,0 +1,32 @@
+<div>
+  HI THERE
```
Comment (losvedir) [inline lib/prediction_analyzer_web/templates/predictions/index.html.eex]: Heh. Maybe this should be an `h1` that says "Recent Predictions" or something like that?

Comment (sghiaa) [reply lib/prediction_analyzer_web/templates/predictions/index.html.eex]: lol i though i deleted that

**Pull request merged**